queue/src/fiskaltrust.Middleware.Localization.QueueDEFAULT/Factories/SignatureItemFactoryDEFAULT.cs

```diff
@@ -6,23 +6,27 @@
 
 namespace fiskaltrust.Middleware.Localization.QueueDEFAULT.Factories
 {
+    // Class responsible for creating signature items for the DEFAULT country setup.
```

Comment:
We should clarify that this is implementation is specific to the default queue and in other markets other things will need to be done.

Maybe also link to Italy for a more advanced example.